### DISCUSSION TITLE
Cannot add records to Budibase db table.

### DISCUSSION THREAD

**Discussion by ndlang** [Budibase/budibase#3130] [Help] [answered]
I am building out my data base using the internal Budibase DB. I am having trouble adding a record to one of the tables. The other tables are behaving as expected. Where would I go to determine the issue?

**Comment by shogunpurple** [comment]
Hi @ndlang - what happens when you add a record to the affected table? 

- Are there any errors in the browser console?
- What is the response from the `/rows` request in the browser when you try to save the row?

**Reply by ndlang** [reply]
Hello @shogunpurple thank you for repsonding here is the error recieved:
![image](https://user-images.githubusercontent.com/17033357/138614440-4fc576d2-2524-4a99-8274-c465758fc3d6.png)

**Reply by shogunpurple** [reply]
Ah, this seems to be the result of a bug that has since been fixed. If you upgrade your budibase installation to the latest version, this should no longer occur.

**Comment by ndlang** [comment, marked as answer]
Yes, updating seemed to fix the issue. Thank you for the suggestion.
